priority_sampler.go

```diff
@@ -15,13 +15,15 @@ type sampleItem struct {
 	// payload:
 	redisJobs       string
```

Comment:
you "could" update the sampling algorithm that actually uses redisJobsPaused. If there are more than 1/2 of the queues that are paused, the task polling is going to be affected.